schema/filter_sale_artworks.js

```diff
@@ -69,9 +42,45 @@ export const filterSaleArtworksArgs = {
   },
 }
 
+export const SaleArtworkAggregations = {
+  description: "Returns aggregation counts for the given filter query.",
+  type: new GraphQLList(SaleArtworksAggregationResultsType),
+  resolve: ({ aggregations }) => {
+    const whitelistedAggregations = omit(aggregations, ["total", "followed_artists"])
+    return map(whitelistedAggregations, (counts, slice) => ({
+      slice,
+      counts,
+    }))
+  },
+}
+
+export const SaleArtworkCounts = {
+  type: new GraphQLObjectType({
+    name: "FilterSaleArtworksCounts",
+    fields: {
+      total: numeral(({ aggregations }) => aggregations.total.value),
+      followed_artists: numeral(({ aggregations }) => aggregations.followed_artists.value),
+    },
+  }),
+  resolve: artist => artist,
+}
+
+export const FilterSaleArtworksType = new GraphQLObjectType({
+  name: "FilterSaleArtworks",
+  fields: () => ({
+    aggregations: SaleArtworkAggregations,
+    counts: SaleArtworkCounts,
+    hits: {
+      description: "Sale Artwork results.",
+      type: new GraphQLList(SaleArtwork.type),
+    },
+  }),
+})
```

Comment:
@alloy - Not sure if I should move the above into `SaleArtworks` now that `FilterSaleArtworks` has been deprecated. Thoughts?

Reply:
Nope, no more plain lists for paginated data.